example_client/stateful/README.md

```diff
@@ -0,0 +1,277 @@
+# OpenVINO™ Model Server Stateful Example Clients 
+
+This document contains examples to run *Predict* functions over gRPC API and REST API on stateful Kaldi models (rm_lstm4f , aspire_tdnn).
+
+It covers following topics:
+* <a href="#grpc-api">gRPC API Stateful Client Example </a>
+* <a href="#rest-api">REST API Stateful Client Example </a>
+
+## Requirement
+
+Install client dependencies using the command below in the example_client directory:
+```
+pip3 install -r stateful_client_requirements.txt
+```
+
+### Getting ready with rm_lstm4f stateful model
+
+To run this example you will need to download the rm_lstm4f model with input and score ark files and convert it to IR format.
+ 1. Download the model from [rm_lstm4f](https://download.01.org/openvinotoolkit/models_contrib/speech/kaldi/rm_lstm4f/)
+ ```bash mkdir models & cd models```
+
+ ```bash wget -r -np -nH --cut-dirs=5 -R *index.html* https://download.01.org/openvinotoolkit/models_contrib/speech/kaldi/rm_lstm4f/ ```
+
+ 2. Convert model to IR [How to convert](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_samples_speech_sample_README.html)
+ 
+ ```bash docker run -u $(id -u):$(id -g) -v $(pwd):/models:rw openvino/ubuntu18_dev:latest deployment_tools/model_optimizer/mo.py --framework kaldi --input_model /models/rm_lstm4f.nnet --counts /models/rm_lstm4f.counts --remove_output_softmax --output_dir /models/rm_lstm4f/1 ```
+
+ 3. Having rm_lstm4f model files .xml and .bin in the IR format present in ```bash $(pwd)/rm_lstm4f/1``` directory,
+    OVMS can be started using the command:
+
+```bash docker run -d --rm -v $(pwd)/rm_lstm4f/:/tmp/model -p 9000:9000 -p 5555:5555 openvino/model_server:latest --stateful --port 9000 --rest_port 5555 --model_name rm_lstm4f --model_path /tmp/model ```
+
+ 4. Return to the example_client/stateful directory
+ ```bash cd .. ```
+
```

Comment:
Is adding `bash` at the beginning of each command necessary?
Also can we increase indentation of the content under each point?